cypress/features/regression/buttonToggleGroup.feature

```diff
@@ -33,27 +33,26 @@ Feature: Button Toggle Group component
   # @ignore because of FE-1447
   # | <>                       |
 
-  @ignore
-  # ignored because the width renders into px
+  @positive
   Scenario Outline: Change Button Toggle Group input width to <width>
-    When I check labelInline checkbox
-      And I set inputWidth to "<width>"
+    Given I check labelInline checkbox
+    When I set inputWidth to "<width>"
     Then input width is set to "<px>"
     Examples:
-      | width | px  |
-      | 1     | 11  |
-      | 10    | 83  |
-      | 100   | 304 |
+      | width | px        |
+      | 1     | 10.75     |
+      | 10    | 107.59375 |
+      | 100   | 825.71875 |
 
   @negative
   Scenario Outline: Set out of scope characters to Button Toggle Group input width to <width>
-    When I check labelInline checkbox
-      And I set inputWidth to "<width>"
+    Given I check labelInline checkbox
+    When I set inputWidth to "<width>"
     Then input width is not set to "<width>"
     Examples:
       | width                   |
       | !@#$%^*()_+-=~[];:.,?{} |
-      | 汉字                    |
+      | 汉字                     |
```

Comment:
we need to use these test cases for inputs (standardized for Storybook)
```
      | title                    |
      | Sample text              |
      | 1234567890               |
      | áéíóú¿¡üñ                |
      | !@#$%^*()_+-=~[];:.,?{}  |
      | ÄÖÜßäöüß                 |

Reply:
Are you saying I need to remove that line altogether leaving only the examples you've shown? I only added a space to line the `|` correctly.

Reply:
Yes, it needs to be just replaced